metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -152,7 +153,11 @@
                    [conn db]
                    (let [conditions [`(>= :id ~start-index)
                                      `(< :id ~(+ start-index batch-size))]
-                         _ (info "Finding batch for provider" provider-id "from id >=" start-index " and id <" (+ start-index batch-size))
+                         _ (info (format "Finding batch for provider [%s] concept type [%s] from id >= %s and id < %s"
```

Comment:
Doug has a dashboard that gets populated from the previous version of this logging statement. Please check with him to verify that this format still matches his regex.